spec/adapters/datatables_spec.rb

```diff
@@ -14,8 +16,22 @@
   end
 
   describe "initialization Tables" do
-    it "Table class must be Daru::DataTables::DataTable " do
+    it "Table class must be Daru::DataTables::DataTable" do
       expect(Daru::View::Table.new.table).to be_a Daru::DataTables::DataTable
     end
+
+    it "Table class must be Daru::DataTables::DataTable when data objects" \
+       " are not of class Array of Arrays" do
```

Comment:
Isn't it better to specifically say what it is rather than "not of class Array of Arrays" ? i.e. "when data is an array"